src/assets/browser/app/src/composit/stage/position.rs

```diff
@@ -48,19 +48,21 @@ impl Position {
         self.zoom.get_screen_in_bp()
     }
     
-    pub fn set_screen_in_bp(&mut self, zoom: f64) {
+    pub(super) fn set_screen_in_bp(&mut self, zoom: f64) {
         self.zoom.set_screen_in_bp(zoom);
     }
     
-    pub fn set_zoom(&mut self, z: f64) {
+    pub(super) fn set_zoom(&mut self, z: f64) {
         self.zoom.set_zoom(z);
+        self.check_own_limits();
     }
     
-    pub fn settle(&mut self) {
+    pub(super) fn settle(&mut self) {
+        return; /* temporary, degrades visual output but buggy */
```

Comment:
Temporarily disable settling (buggy) to be re-enabled in a later PR (probably). If not code should be deleted.